x-pack/plugins/infra/server/lib/adapters/log_entries/kibana_log_entries_adapter.ts

```diff
@@ -266,7 +266,7 @@ const convertHitToLogEntryDocument = (fields: string[]) => (
             [fieldName]: get(fieldName, hit._source),
           }
         : flattenedFields,
-    {} as { [fieldName: string]: string | number | boolean | null }
+    {} as { [fieldName: string]: string | number | null }
```

Comment:
What's the reasoning behind the removal of `boolean` here?

Reply:
Here is the error that it throws when I add `boolean` back in: 

```
Type '{ [fieldName: string]: string | number | boolean | null; }' is not assignable to type 'LogEntryDocumentFields'.
  Index signatures are incompatible.
    Type 'string | number | boolean | null' is not assignable to type 'string | number | null'.
      Type 'false' is not assignable to type 'string | number | null'.
```

Reply:
Looks like `boolean` is missing from `LogEntryDocumentFields`